core_lib/src/structure/layercamera.cpp

```diff
@@ -589,11 +588,17 @@ void LayerCamera::loadDomElement(const QDomElement& element, QString dataDirPath
                 CameraEasingType easing = static_cast<CameraEasingType>(imageElement.attribute("easing", "0").toInt());
                 qreal pathX = imageElement.attribute("pathCPX", "0").toDouble();
                 qreal pathY = imageElement.attribute("pathCPY", "0").toDouble();
-                bool pathMoved = imageElement.attribute("pathCPM", "0").toInt();
+                bool pathMoved = (imageElement.hasAttribute("pathCPX") || imageElement.hasAttribute("pathCPY")) &&
+                                 imageElement.attribute("pathCPM", "1").toInt(); // BC
 
                 loadImageAtFrame(frame, dx, dy, rotate, scale, easing, QPointF(pathX, pathY), pathMoved);
             }
         }
         imageTag = imageTag.nextSibling();
     }
+    foreachKeyFrame([&](KeyFrame* frame) {
```

Comment:
wouldn't it be better to do this in the loadIamgeAtFrame rather than looping through all frames again?

Reply:
Ah... you're right, my bad.. i always forget that we store the data flipped of what we're seeing.. so it starts loading frame 10 then frame 9.. that's of course an issue in regard to that...

I'm all for simplicity so we could look into that.. it's a fundamental change but that doesn't mean we can't change it... we're still in dev mode :P